src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/PatchCorrector.kt

```diff
@@ -18,19 +18,54 @@ import java.util.LinkedList
 import name.fraser.neil.plaintext.diff_match_patch
 
 object PatchCorrector {
+    private val dmp = diff_match_patch()
+
+    private fun textToPatches(patchList: List<CosmasProto.Patch>) : LinkedList<diff_match_patch.Patch> {
+        val newPatchList = LinkedList<diff_match_patch.Patch>()
+        for (patch in patchList) {
+            newPatchList.addAll(textToPatch(patch))
```

Comment:
Can't help thinking that replacing `textToPatch(patch)` with `dmp.patch_fromText(patch.text)` will make this code more clear and efficient. Perhaps the other usage of `textToPatch` can be replaced with the same call and that method will no longer be needed?

Reply:
I want this method to return LinkedList, not MutableList, for use it with dmp without conversion MutableList to LinkedList

Reply:
`LinkedList(dmp.patch_fromText(deleteCandidate.text)`

Reply:
delete function